internal/jobcontainers/path.go

```diff
@@ -0,0 +1,57 @@
+package jobcontainers
+
+import (
+	"errors"
+	"os"
+	"os/exec"
+	"path/filepath"
+)
+
+// Checks the sandbox volume and PATH env variable to find application name. If the path
+// is relative, E.g. path\to\binary, it will be searched for in the sandbox volume. If it
+// is solely an application name then the base directory of the sandbox volume and all directories
+// in PATH will be searched. Job containers can run anything on the host so C:\path\to\exe,
+// D:\path\to\exe etc. are just passed through as is.
+func findExecutable(path string, imagePath string) (string, error) {
```

Comment:
We should probably just follow the same logic as CExecSvc uses here. Having the same behavior would reduce confusion.

Reply:
Done!